src/Apps/ArtAdvisor/07-Curated-Discovery/Components/Result/ArticlesRail.tsx

```diff
@@ -0,0 +1,61 @@
+import { Box, SkeletonBox } from "@artsy/palette"
+import { State } from "Apps/ArtAdvisor/07-Curated-Discovery/App"
+import { Article } from "Apps/ArtAdvisor/07-Curated-Discovery/Components/Result/Article"
+import { Rail } from "Components/Rail/Rail"
+import { FC, useEffect, useState } from "react"
+
+interface ArticlesRailProps {
+  state: State
+}
+
+export interface ArticleType {
+  id: string
+  imageUrl?: string
+  href: string
+  title: string
+  articleDescription: string
+}
+
+export const ArticlesRail: FC<ArticlesRailProps> = ({ state }) => {
+  const [articles, setArticles] = useState<ArticleType[]>([])
+  const [isLoading, setIsLoading] = useState<boolean>(true)
+
+  useEffect(() => {
+    const params = new URLSearchParams()
+    params.append("limit", "25")
```

Comment:
Ah, I think I answered  my own question (just to confirm, that is the reason for the higher limit here yes?)

Reply:
Yup! exactly.